{{cookiecutter.github_project_name}}/package.json

```
@@ -49,33 +49,34 @@
     "watch:nbextension": "webpack --watch"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.1.10 || ^2"
+    "@jupyter-widgets/base": "^2.0.2"
   },
   "devDependencies": {
-    "@phosphor/application": "^1.6.0",
-    "@phosphor/widgets": "^1.6.0",
+    "@phosphor/application": "^1.7.3",
+    "@phosphor/widgets": "^1.9.3",
     "@types/expect.js": "^0.3.29",
-    "@types/mocha": "^5.2.5",
-    "@types/node": "^10.11.6",
-    "@types/webpack-env": "^1.13.6",
+    "@types/mocha": "^7.0.1",
+    "@types/node": "^13.7.0",
+    "@types/webpack-env": "^1.15.1",
     "css-loader": "^3.2.0",
```

We might want to add babel polyfill

```suggestion
    "css-loader": "^3.2.0",
    "babel-polyfill": "^6.26.0",
```